server.mjs

```diff
@@ -0,0 +1,15 @@
+import express, { static as expressStatic } from 'express';
+import { join, resolve } from 'path';
+const PORT = process.env.PORT || 5000;
+
+const app = express();
+
+app.use(expressStatic(join(resolve(), 'dist')));
+
+app.all('*', (_, res) => {
+  return res.sendFile(join(resolve(), 'dist/index.html'));
+});
+
+app.listen(PORT, () => {
+  console.log('App being served on port: ', PORT);
+});
```

Comment:
Why do we need a common js server file ?

Reply:
Since on heroku we cannot use the `webpack` built pack to run our application without a performance cost **(Installing webpack as a dependency)**, this file configure express to serve the static folder and redirect all other traffics to our application.

Reply:
The need for a `.mjs` is for that purpose and that purpose alone. The extension is to allow usage of `import` without the need to speciy the `type=module` in the `package.json`